From d2d827b767047d3f01618d066fa5bb10923b0a44 Mon Sep 17 00:00:00 2001
From: crloz <53350353+crloz@users.noreply.github.com>
Date: Sat, 20 Nov 2021 12:21:02 -0500
Subject: [PATCH] remove smartcard and fingerprint

---
 data/meson.build | 2 --
 1 file changed, 2 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index 2dec4c23..4b131473 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -127,8 +127,6 @@ pam_data_files_map = {
   'arch': [
     'gdm-autologin',
     'gdm-launch-environment',
-    'gdm-fingerprint',
-    'gdm-smartcard',
     'gdm-password',
   ],
   'none': [],
-- 
2.33.0

